src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.tsx

```diff
@@ -153,6 +182,75 @@ export const TrackPanelList = () => {
   );
 };
 
+type VariantGroups = VariantGroup[];
+
+type VariantGroup = {
+  id: number; // same as the numbers used in genome browser data files to categorise variants
+  label: string;
+  variant_types: VariantType[];
+};
+
+type VariantType = {
+  label: string;
+  so_accession_id: string;
+  url: string;
+};
+
+const VariantGroupsLegend = (props: {
```

Comment:
This component could easily have been defined outside of TrackPanelList component.